lib/LaTeXML/Post.pm

```diff
@@ -447,7 +447,7 @@ sub convertXMTextContent {
   foreach my $node (@nodes) {
     if ($node->nodeType == XML_TEXT_NODE) {
       my $string = $node->textContent;
-      $string =~ s/^\s+/$NBSP/; $string =~ s/\s+$/$NBSP/;    # should we???
```

Comment:
All the leading and ending spaces XMText are substituted by one non-breaking space respectively, I suppose Bruce wants to reduce the verbosity here and have a nicer output. 

Meanwhile Michael reported
```tex
&#xA0;and&#xA0;
```
is causing troubles in sTeX.

I don't really see other aspects of having to retain this regular expression, but please correct me if I am wrong. 

Or we can try to change the pattern into something else, so that sTeX doesn't complain?